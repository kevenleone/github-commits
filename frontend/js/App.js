import React from 'react';
import { hot } from 'react-hot-loader';
import { ToastContainer } from 'react-toastify';
import { Provider } from 'react-redux';

import { Home } from './pages/Home';
import store from './redux/store';
import SentryBoundary from './utils/SentryBoundary';
import 'react-toastify/dist/ReactToastify.css';
import Load from './components/Load/Load';

const App = () => (
  <SentryBoundary>
    <Provider store={store}>
      <ToastContainer />
      <Load />
      <Home />
    </Provider>
  </SentryBoundary>
);

export default hot(module)(App);
