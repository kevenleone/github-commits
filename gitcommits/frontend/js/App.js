import React from 'react';
import { hot } from 'react-hot-loader';
import { ToastContainer } from 'react-toastify';

import { Home } from './pages/Home';

import 'react-toastify/dist/ReactToastify.css';
import SentryBoundary from './utils/SentryBoundary';

const App = () => (
  <SentryBoundary>
    <ToastContainer />
    <Home />
  </SentryBoundary>
);

export default hot(module)(App);
