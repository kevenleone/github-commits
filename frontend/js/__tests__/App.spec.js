import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('App', () => {
  test('should render with success', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
