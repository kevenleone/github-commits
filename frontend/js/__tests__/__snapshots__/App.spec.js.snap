// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render with success 1`] = `
<ExampleBoundary>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Header />
    <ToastContainer
      autoClose={5000}
      bodyClassName={null}
      className={null}
      closeButton={
        <CloseButton
          ariaLabel="close"
        />
      }
      closeOnClick={true}
      draggable={true}
      draggablePercent={80}
      hideProgressBar={false}
      newestOnTop={false}
      pauseOnFocusLoss={true}
      pauseOnHover={true}
      position="top-right"
      progressClassName={null}
      progressStyle={null}
      role="alert"
      rtl={false}
      style={null}
      toastClassName={null}
      transition={[Function]}
    />
    <Load />
    <Routes />
  </Provider>
</ExampleBoundary>
`;
